app/controllers/avo/associations_controller.rb

```diff
@@ -80,7 +80,7 @@ def create_association
       association_name = BaseResource.valid_association_name(@record, association_from_params)
 
       perform_action_and_record_errors do
-        if through_reflection? && additional_params?
+        if through_reflection? && !additional_params.empty?
```

Comment:
```suggestion
        if through_reflection? && additional_params.any?
```

Reply:
There's no `any?` for `ActionControl::Params`. I could do `additional_params.to_h.any?` WDYT?